src/presentation/ui/page/databases.templ

```diff
@@ -0,0 +1,299 @@
+package page
+
+import (
+	"github.com/speedianet/os/src/domain/entity"
+	"github.com/speedianet/os/src/domain/valueObject"
+	infraHelper "github.com/speedianet/os/src/infra/helper"
+	componentDatabases "github.com/speedianet/os/src/presentation/ui/component/databases"
+	componentForm "github.com/speedianet/os/src/presentation/ui/component/form"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+	presenterDto "github.com/speedianet/os/src/presentation/ui/presenter/dto"
+)
+
+script DatabasesIndexLocalState() {
+    document.addEventListener('alpine:init', () => {
+		Alpine.data('databases', () => ({
+			// Primary states
+			database: {},
+			databaseUser: {},
+			setDatabaseUserPassword(password) {
+				this.databaseUser.password = password;
+			},
+			resetPrimaryStates() {
+				this.database = {
+					'name': '',
+					'size': ''
+				}
+				this.databaseUser = {
+					'username': '',
+					'password': '',
+					'privileges': []
+				}
+			},
+			init() {
+				this.resetPrimaryStates();
+			},
+
+			// Auxiliary states
+			changeSelectedDatabaseType(databaseType) {
+				htmx.ajax(
+					'GET',
+					'/databases/?dbType=' + databaseType,
+					{
+						select: '#databases-content',
+						target: '#databases-content',
+						swap: 'outerHTML transition:true'
+					},
+				);
+			},
+			installDatabaseService(databaseType) {
+				htmx.ajax(
+					'POST',
+					'/api/v1/services/installables/',
+					{
+						swap: 'none',
+						values: { name: databaseType },
+					},
+				);
+			},
```

Comment:
Interesting, why isn't this in the html? Can't you use hx-post? Weird. Let's see.

Reply:
This was already discussed on Slack, and we agreed on this code.